### ISSUE TITLE
Can't open ODF file from command line

### ISSUE THREAD

**Issue by limelime** [antoniofaccioli/libreoffice-appimage#12]
When executed the following command:
```bash
LibreOffice-fresh.basic-x86_64.AppImage file.odt
```
it showed the following error:
```
/tmp/.mount_LibreOEGfULE/usr/file.odt does not exist.
```

However, opening file.odt from Libreoffice GUI is fine.

**Comment by probonopd** [comment]
The `AppRun` which is currently being used does a chdir into `usr/` of the AppImage before it executes the payload application. This means passing in relative paths does not work.

https://github.com/AppImage/AppImageKit/blob/d18552660f2823102b6da4b0b61821e9bf8feb02/src/AppRun.c#L155-L161

If you want relative paths to work, you currently need to use your own version of `AppRun`. A simple symlink to the script that launches LibreOffice may be sufficient (though I have not tested it).

**Comment by antoniofaccioli** [comment]
I have tested it on Zorin OS 12 (derivative of Ubuntu 16.04), if you give it the absolute path of the file it works correctly.

For example:  LibreOffice-fresh.basic-x86_64.AppImage /home/antonio/Scrivania/Esercitazioni.odt

**Comment by probonopd** [comment]
Yes. If we want to make it support relative paths, we need to use a different `AppRun`.

**Comment by limelime** [comment]
@antoniofaccioli 
Yes, absolute path can be used to open the file but using Libreoffice to do conversion will create same problem.
e.g
```
LibreOffice-fresh.basic-x86_64.AppImage --headless --convert-to docx:"MS Word 2007 XML"  /home/absolute/path/file.odt 
convert /home/absolute/path/file.odt -> /tmp/.mount_libreoFayDko/usr/file.docx using filter : MS Word 2007 XML
Error: Please verify input parameters... (SfxBaseModel::impl_store <file:///tmp/.mount_libreoFayDko/usr/file.docx> failed: 0x507(Error Area:Io Class:Access Code:7))
```
In order to resolve this, I have to explicitly specify the output directory using `--outdir`.
